ddtrace/internal/debug.py

```diff
@@ -134,4 +134,6 @@ def collect(tracer):
         global_tags=os.getenv("DD_TAGS", ""),
         tracer_tags=tags_to_str(tracer.tags),
         integrations=integration_configs,
+        partial_flush_enabled=tracer._partial_flush_enabled,
+        partial_flush_min_spans=tracer._partial_flush_min_spans,
```

Comment:
I don't love reading this private property... but it does seem the easiest way to do it

Reply:
Don't think we have an alternative here since it is an internal + undocumented API. This one isn't too bad all things considered